netsuite.gemspec

```diff
@@ -17,5 +17,5 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency 'savon', '>= 2.3.0'
 
-  gem.add_development_dependency 'rspec', '~> 3.1.0'
+  gem.add_development_dependency 'rspec', '~> 3.12.0'
```

Comment:
What is your heuristic for bumping the gem version? Any change to the gem? Any change which requires dependency changes? Or any change which requires a new version of Ruby?

Reply:
Any changes that depends on changing rules of gems should be changed their version, this makes easier for you to check if a gem was bumped for a project and easily revert the change if need be. Maintaining the current version of a gem makes reversion harder, as now you have to point to a particular commit of the gem, rather than its version.

There's a brief explanation of how we normally view gems versioning here
https://guides.rubygems.org/patterns/#semantic-versioning

Reply:
Right now, as we're referencing it from Konekti, we're referring to a branch.
![image](https://github.com/mavenlink/netsuite/assets/1417659/ea26c0b3-54f3-4413-abb9-3b88be51c888)

We should bump the version in the gem and change how we're referencing it from Konekti to be less psychotic. 
![image](https://github.com/mavenlink/netsuite/assets/1417659/fa6f59de-aaa2-4951-9a4e-5fe36fede26c)

Reply:
Would this be considered patch, minor or major? I guess does the consumer need to change? I don't think so, so maybe patch number + 1?